e2e/windows/e2e-helper.sh

```diff
@@ -16,8 +16,22 @@ exec_on_host() {
     kubectl exec $(kubectl get pod -l app=debug -o jsonpath="{.items[0].metadata.name}") -- bash -c "nsenter -t 1 -m bash -c \"$1\"" > $2
 }
 
+backfill_clean_storage_container() {
+    set +x
+    # list outdated storage containers created over 1 week ago
```

Comment:
@junjiezhang1997 This will only delete those containers created over 1 week ago so it should not impact our tests/builds, is my understanding right?

Reply:
I don't think so. The containers are reused by different pipelines. In my opinion, we should delete container whose kubernetes version is not supported any more.

Reply:
You are saying there may be problems running concurrent builds right? E.g. if pipeline A runs this function to clean something say the container for 1.24.9 that was created over a week ago, another concurrent pipeline B thought the container existed only to find it was later deleted by pipeline A?

How should we pass obsolete kubernetes version? Pipeline variables? Hard-coded?

Would it be better if we add a variable like "BACKFILL_CONTAINER" such that only those pipelines with this switch turned on will clean up - no conflict any more?

Reply:
We don't need to pass obsolete kubernetes versions. We can use az cli to get supported kubernetes versions and check whether the version of container is still supported.

For example: az aks get-versions -l eastus --query "orchestrators[?default==\`true\`].orchestratorVersion" -otsv

Reply:
The variable like "BACKFILL_CONTAINER" will not work. As long as a container with supported version is deleted and the cluster with the same supported version is not deleted yet, a new pipeline will fail to get linux file from the deleted container.